README.md

```diff
@@ -10,12 +10,18 @@ This repository contains specifications for the `mx-sdk-*` libraries. The specif
 - `abi`: ABI components and ABI-aware codecs.
 - `adapters`: components that resolve the impedance mismatch between interfaces of different domains.
 - `converters`: components that are able to convert concepts (structures, classes) between different domains.
+- `account-management`: contains all files related to the account management interactions (controller,factory, parser, entities).
```

Comment:
Not sure we should keep the `Account` entities in this package. I think we should have an `accounts` package that contains `Account`, `LedgerAccount`, `GuardianAccount`, etc.